### ISSUE TITLE
Should builtin_item be listed as a dependency

### ISSUE THREAD

**Issue by jfanjoy** [joe7575/techage#167]
Even though there is no technical dependency on the builtin_item mod, I don't think the gravel rinser items will move with flowing water without it. The rinser will work, but collecting items using a downstream hopper may not.

**Comment by joe7575** [comment]
> Should builtin_item be listed as a dependency

I don't believe. They are built-in items, as the name suggests

> Even though there is no technical dependency on the builtin_item mod, I don't think the gravel rinser items will move with flowing water without it.

It must an item_group from the type "water" and it must have the liquidtype  "flowing".  
See: https://github.com/joe7575/techage/blob/master/basic_machines/gravelrinser.lua#L78

**Comment by jfanjoy** [comment]
I did eventually get the rinser working (kudos to you on a really creative piece of machinery). However it seems at least in my testing that default minetest water behavior does not push entities and so the items found by the rinser weren't making it to the hopper. The [builtin_item](https://content.minetest.net/packages/TenPlus1/builtin_item/) mod seems less to do with any items and more to do with the way items interact with water, ice, and lava.

As for the flowing water I did find that the param2 value for water flowing in a downward direction is greater than the param2 value for water flowing in a horizontal direction. My initial configuration had the hopper at 1 block below the rinser which didn't work even though the water was flowing because of the param2 difference.
